api/e2e/test/helpers_bigquery_test.go

```diff
@@ -17,6 +17,8 @@ import (
 //
 // If at least <minRowCount> rows is returned, the no of rows retrieved will be returned.
 // Else if timeout is reached, error will be non nil.
+//
+// nolint:deadcode,unused
```

Comment:
Same comment as above - can we simply remove this? The e2e folder is replicated internally anyway.